extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/JDBCEnvironment.java

```diff
@@ -67,6 +67,19 @@ public JDBCEnvironment() {
      */
     public abstract int getAbsoluteMaxConnections() throws GuacamoleException;
 
+    /**
+     * Returns the default maximum number of identifiers/parameters to be 
+     * included in a single batch when executing SQL statements.
+     *
+     * @return
+     *     The default maximum number of identifiers/parameters to be included 
+     *     in a single batch.
+     *
+     * @throws GuacamoleException
+     *     If an error occurs while retrieving the property.
+     */
+    public abstract int getDefaultBatchSize() throws GuacamoleException;
```

Comment:
Is it the default batch size? Or is it just the batch size?

Reply:
Yeah it's just the batch size